winui/Avatar-view/how-to.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: States in WinUI AvatarView control | Syncfusion
+description: Learn about how to set the different states of badge for the WinUI AvatarView control(SfAvatarView).
```

Comment:
@MadhumithanJayapathi Change the title "Badge in WinUI AvatarView control | Syncfusion"

Reply:
@Manivannan-E Changed the title as given.